dev/build_update.py

```diff
@@ -141,7 +141,7 @@ def build_update_file(
     meta_data = {
         "update_file_format": "1.0",
         "supported_hardware": ["vector_v4", "vector_v5"],
-        "micropython_versions": ["1.24.1"],
+        "micropython_versions": ["1.24.1", "1.23.0-preview"],
```

Comment:
Since this is currently working on 1.23.0-preview I've added that back to the supported versions. Eventually we'll need to remove it because it's memory management seems to be worse, but for the moment it seems fine.